operator/src/main/java/oracle/kubernetes/operator/helpers/CrdHelper.java

```diff
@@ -114,8 +111,7 @@ static V1beta1CustomResourceDefinition createModel(KubernetesVersion version) {
 
     static V1ObjectMeta createMetadata() {
       return new V1ObjectMeta()
-          .name(KubernetesConstants.CRD_NAME)
-          .putLabelsItem(LabelConstants.RESOURCE_VERSION_LABEL, DEFAULT_OPERATOR_VERSION);
```

Comment:
why are these labels removed?

Reply:
that might be branch/merge issue.

Reply:
I removed them, but I can move this to a different PR.

Reply:
Long ago, we thought we would use these labels to know when to update the CRD... Now, we use the CRD's version field, so we don't need these labels.  They just cause confusion.